Enterprise-CMCS/cmcs-eregulations
# EREGCSC-1816 -- a few Vue 3 upgrade milestones

Resolves [EREGCSC-1816](https://jiraent.cms.gov/browse/EREGCSC-1816)

**Description**

This PR will upgrade Vuetify to 2.7, the LTS release.  This will be the final Vuetify 2 release and will be somewhat of a bridge between Vuetify 2 and Vuetify 3.

**This pull request changes**

- Upgrade to Vuetify 2.7

**Steps to manually verify this change:**

1. Visit experimental deployment and make sure everything looks fine
2. All tests pass
